lib/osut/utils.rb

```diff
@@ -4673,37 +4844,18 @@ def genInserts(s = nil, set = [])
       end
     end
 
-    # Loop through each 'valid' set (i.e. linking a valid leader line anchor),
-    # generate set vertex array based on user-provided specs. Reset BLC vertex
-    # coordinates once completed.
-    set.each_with_index do |st, i|
-      str = id + "set ##{i+1}"
-      dZ  = nil
-      t   = nil
-      bx  = st[:box]
-
-      if facingUp?(bx)
-        if xyz?(bx, :z)
-          dZ = 0
-        else
-          dZ = bx.first.z
-          bx = flatten(bx).to_a
-        end
-      else
-        t  = OpenStudio::Transformation.alignFace(bx)
-        bx = t.inverse * bx
-      end
-
-      o = getRealignedFace(bx)
-      next unless o[:set]
-
-      st[:out] = o
-      st[:bx ] = blc(o[:r] * (o[:t] * o[:set]))
```

Comment:
_Bounded boxes_ are systematically bottom-left-cornered (BLC) - no longer any need to post-process them all.